eb_hooks.py

```diff
@@ -199,6 +199,15 @@ def parse_hook_ucx_eprefix(ec, eprefix):
         raise EasyBuildError("UCX-specific hook triggered for non-UCX easyconfig?!")
 
 
+def parse_hook_xvfb_cp_permissions(ec, eprefix):
+    """Add user write permission to xvfb-run before copying it in the postinstallcmds."""
+    if ec.name == 'Xvfb':
+        ec['postinstallcmds'] = ["chmod u+wx xvfb-run && cp -a xvfb-run %(installdir)s/bin/"]
```

Comment:
@TopRichard Can you clarify why this is needed?

If there's a problem here, I'm not sure why we didn't see it outside of EESSI...

Reply:
@boegel Xvfb is a dependency, and so it was set to avoid a failure.

Reply:
I got that part, but I don't see why we would see this problem with Xvfb outside of EESSI.

I'm seeing it too in #328 where I'm trying to get `R` installed (just to break down this PR a bit), so I'll take a closer look.

Reply:
@boegel I debugged that issue some time ago, see https://github.com/EESSI/software-layer/pull/299#issuecomment-1629170830 and the messages below that one. TLDR (if I remember correctly): it's a combination of our writable overlay + setting the installation dir as read-only (and `xvfb-run` is shipped as "patch" in the EB installation dir).

Reply:
I'm also running into the same issue with https://github.com/EESSI/software-layer/pull/334

Reply:
@boegel Could the issue with Xvfb be related to the version of XZ? That is the only difference I can discern in EESSI.

Reply:
It's (I believe) somehow related to the fact we now have write-only installations and the writable overlay. We should probably create an upstream PR to reverse the order of statements in the `Xvfb` `postinstallcmds`, which is what the hook update here is doing (or we just move forward with the hook update here).